ParabolInc/action
# stored xss fix

usually i'd quietly merge security things, but this is such an edge case I don't mind making it public.
this also gets us 1 step closer to removing redux-form, which will unlock the ability to migrate to the new react

TEST

- [ ] can upload new user profile pic
- [ ] can upload new org profile pic
- [ ] if i upload a svg with a script tag in it, i see an easter egg

to reproduce the last step:
- open up any of our svg images (eg mark-white.svg)
-  on the 2nd to last line add a tag like:
```
<script>
        console.log('HACKED');
    </script>
```
- save the file & then try to upload it
